### PR TITLE
Inventory changing

### LINKED ISSUE #50: Cannot apply inventory changes
Open inventory, press `A` =>

```
r.o.c.k.s.g.InventoryCommand: Inventory changed: Inventory(Set(Leg(right leg,75,Set(WalkCapability)), Arm(left arm,50,Set(ManipulatorCapability)), Leg(left leg,75,Set(WalkCapability)), Arm(right arm,50,Set(ManipulatorCapability)), Head(head,75,Set()), Torso(torso,100,Set())),Set(Knife(Knife,100)),Set())
r.o.c.k.s.i.InventoryScene: render
Uncaught java.lang.IllegalStateException: Promise already completed.
```

That's not a good thing.

### PR DESCRIPTION
**Pull request by ForNeVeR**
It enables Dr. Növer to wield his old good knife again, and maybe even slay some monsters or people with it. Closes #50.

### CODE REVIEW

File: src/test/scala/ru/org/codingteam/keter/game/actions/ChangeInventoryActionTest.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import ru.org.codingteam.keter.game.objects.equipment.bodyparts.Head
+import ru.org.codingteam.keter.game.objects.equipment.items.Knife
+import utest._
+
+object ChangeInventoryActionTest extends TestSuite {
+  val universe = Location.createLocation()
+  val action = InventoryChangeAction(Inventory(Set(Head("h", 10)), Set(Knife("k")), Set()))
+
+  val tests = TestSuite {
+    'shouldChangeInventory {
```
Comment (rexim) [inline src/test/scala/ru/org/codingteam/keter/game/actions/ChangeInventoryActionTest.scala]: Is this some kind of Lisp quotation?

Comment (ForNeVeR) [inline src/test/scala/ru/org/codingteam/keter/game/actions/ChangeInventoryActionTest.scala]: Yeah, that's a symbol. Analogous to lisp keyword in practice.

Comment (rexim) [inline src/test/scala/ru/org/codingteam/keter/game/actions/ChangeInventoryActionTest.scala]: Wow, didn't know about those. Thanks!

**Comment by rexim** [comment]
Ok, the shit has been approved!

**Pull request merged**